packages/web/src/app/doc-pages/components/tag-doc/tag-signal-colored-ceg/tag-signal-colored-ceg.component.html

```diff
@@ -0,0 +1,4 @@
+<div class="e-tag e-tag--green">Ny</div>
+<div class="e-tag e-tag--red">Farlig</div>
+<div class="e-tag e-tag--orange">Advarsel</div>
+<div class="e-tag e-tag--neutral">Info</div>
```

Comment:
If these are Signal colors, shouldn't they be `e-tag--positive`, `--warning`, `--error`?

Reply:
I suppose they can be, however it also feels a bit context dependent. 
For example `--error` doesn't necessarily mean the same as a tag with "Farlig", even though they are both the signal red 🤔 We have `--error`, `--danger`, `--red` that all do the same signal red color, and similarily several names for the same other colors. 
![image](https://github.com/3lvia/designsystem/assets/38653456/088c44d5-bdb9-4b0d-aa9d-6f4e5c329003)

Reply:
Ah, I see what you mean by it being context-dependent.

Personally, I think it would be best to use `-positive`, `-caution`, `-warning`, `-danger`, and `-info` to align with the _Signal_-section on the [Colors-documentation](https://deploy-preview-2942--elvis-designsystem.netlify.app/brand/color#How%20to%20use%20tokens), but the current implementation works too.